bin/speclj

```diff
@@ -0,0 +1,20 @@
+#! /usr/bin/env phantomjs
```

Comment:
Could just be me not understanding - but not sure why you're running Phantom JS. Can't you just run the test suite from the command line?

Reply:
I set up my project using `lein new app name-of-app` (if I remember correctly) and it came with a setup for ClojureScript, so I'm guessing this is a leftover that I will remove!